tox.ini

```diff
@@ -30,6 +30,7 @@ commands =
   /bin/bash -c 'py.test -vvvv -n `python -c "import multiprocessing; import os; print(int(os.environ.get(\"XDIST_CPU\") or multiprocessing.cpu_count()))"` --dist=loadfile --maxfail=15 --durations=100 --cov ansible_navigator --cov share --cov-report term-missing --cov-branch --showlocals'
 setenv =
   TERM = xterm-256color
+  COVERAGE_FILE = {env:COVERAGE_FILE:{toxworkdir}/.coverage.{envname}}
```

Comment:
I don't think this is necessary. I suppose you may be looking for an XML report. In which case, add `--cov-report xml` maybe?

Reply:
Not needed for codecov itselt but it helps decluttering project root, so we don't add extra temp files to it.

Reply:
Removed it as is not directly needed by codecov.

Reply:
It is, unless you configure it to use a different report format IIRC